ansible/roles/ansible-datamover-api/tasks/create_dmapi_db.yml

```diff
@@ -1,12 +1,29 @@
-- import_tasks: db_setup.yml
+- import_tasks: db_setup_victoria.yml
   run_once: True
   vars:
     _oslodb_setup_host: "{{ dmapi_db_setup_host }}"
+    _oslodb_ansible_python_interpreter: "{{ dmapi_db_setup_python_interpreter }}"
```

Comment:
can we not set the values directly here instead of defining dmapi_db_setup_host and dmapi_db_setup_python_interpreter in a separate variables yaml file ?

Reply:
or we can add same code in tasks/main.yml file  instead of separate variables yaml file.